projects/packages/connection/src/sso/class-sso.php

```diff
@@ -510,7 +503,7 @@ public function login_init() {
 			 * to the WordPress.com login page AND  that the request to wp-login.php
 			 * is not something other than login (Like logout!)
 			 */
-			if ( ! $this->use_wp_admin_interface() && Helpers::bypass_login_forward_wpcom() && $this->wants_to_login() ) {
```

Comment:
We previously disabled it entirely for WP-Admin classic sites, right? 🤔

Reply:
Correct. Now we allow it if the user comes from Calypso, and keep it disabled otherwise.